src/datepicker/bs-datepicker.module.ts

```diff
@@ -50,22 +37,35 @@ const _exports = [
     BsCurrentDateViewComponent,
     BsDatepickerNavigationViewComponent,
     BsTimepickerViewComponent,
-
     BsCalendarLayoutComponent,
     BsDaysCalendarViewComponent,
     BsMonthCalendarViewComponent,
     BsYearsCalendarViewComponent,
-
     BsCustomDatesViewComponent,
-
-    ..._exports
```

Comment:
Did you open an issue against ngcc with Ivy to support the spread operator?

Reply:
it *already fixed*, I checked it on angular 8.1.2
https://github.com/angular/angular/issues/29835